lifecycle_policy.md

```diff
@@ -15,38 +15,74 @@ Capitalized terms not otherwise defined in this Project Lifecycle Policy have th
+
+To see past project proposals, you can look at the closed issues in this repository. See here:
+
+* [Emerging Project Proposals](https://github.com/hpsfoundation/tac/issues?q=is%3Aissue%20state%3Aclosed%20label%3Alevel%3Aemerging)
+* [Established Project Proposals](https://github.com/hpsfoundation/tac/issues?q=is%3Aissue%20state%3Aclosed%20label%3Alevel%3Aestablished)
+* [Core Project Proposals](https://github.com/hpsfoundation/tac/issues?q=is%3Aissue%20state%3Aclosed%20label%3Alevel%3Acore)
 
 ### Project Acceptance Process
-* Projects are required to present their proposal at a TAC meeting
-* The TAC may ask for changes to bring the project into better alignment with the High Performance Software Foundation (adding a governance document to a repository or adopting a Code of Conduct, for example).
-* The project will need to make these changes in order to progress further.
-* Projects are accepted by a 2/3 majority of the TAC.
-* Vote majorities will be calculated as a percentage of the votes received
-* The TAC will determine the appropriate initial stage for the project. The project can apply for a different stage via the Periodic Review Process.
+Once a project has submitted a proposal as outlined above, the process for accepting them
+to HPSF is as follows:
+
+1. TAC sponsors shall read through the project proposal and ensure that all criteria
+   for the project's requested level are addressed in the proposal. If insufficient
+   evidence is presented for any of the criteria, the sponsors should work with the
+   project to bolster the case, or to choose a different, more appropriate initial
+   level.
+
+2. Once the proposal is in a suitable state, the sponsors shall propose it to the full
+   TAC for onboarding, and the TAC shall vote on whether to hear the project's presentation.
+
+3. If the TAC votes to take up the proposal, the TAC shall decide on a future meeting to
+   hear the presentation.
+
+4. TAC voting members shall read the project's proposal and ensure that they are familiar
+   with the state of the project to be onboarded *before* the onboarding presentation.
+
+5. Project members shall present their proposal at the scheduled TAC meeting. This should take
+   no more than 30 minutes, and it will typically involve an introduction to the project and
+   a verbal explanation of alignment with HPSF and key elements of the proposal.
+   See links in the prior section to past project propsoals for links to past talks.
+
+6. Once the project members have presented, voting members of the TAC shall convene a
+   private session, and the project's sponsors shall discuss the project, its presentation,
+   its proposal, and any questions the voting members have as to the evidence for
+   onboarding criteria.
+
+   In the closed sessio, sponsors are expected to advocate for the projects to be
+   onboarded and to clarify any questions the TAC may have.
```

Comment:
```suggestion
   In the closed session, sponsors are expected to advocate for the project and to clarify any questions the TAC may have.
```
Applying multiple suggestions editing the the same line is not supported so I consolidated https://github.com/hpsfoundation/tac/pull/45/files#r2114260919, https://github.com/hpsfoundation/tac/pull/45/files#r2135676375 and https://github.com/hpsfoundation/tac/pull/45/files#r2135676679